plugin/helpers.go

```diff
@@ -74,3 +83,28 @@ func (p *HelpersImpl) ensureServerVersion(required string) error {
 	}
```

Comment:
I just noticed that you are the implementation is also part of `plugin/helpers.go`. I would like to keep this file with just the interface. Would you mind moving the implementation to `plugin/helpers_plugin.go`?

Reply:
I will be making the changes.